### DISCUSSION TITLE
Obsidian中使用Vim的最佳实践？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#10] [实用技巧/最佳实践]
obsidian中使用vim有哪些坑？有什么好的方法

**Comment by makerjackie** [comment]
你需要通过插件或者其他方式解决一些vim的刚需

插件（在obsidian plugin中搜索）
* Vimrc Support：支持vimrc，有几个重要的功能，复制到系统剪贴板，map j -> gj
* add-codemirror-matchbrackets：支持使用 `di(` （跟`dt(`的区别在 `di(` 删除到对应匹配的括号，`dt(` 则是删除到第一个遇到的`(`）

中英文切换问题

我目前是通过autohotkey来解决的，参考：https://www.zhihu.com/question/303850876/answer/1181682863

**Reply by LeafLight** [reply]
请问`.obsidian.vimrc`是放在`/VultrName`下吗，我放在这目录下似乎没有用

**Comment by Quorafind** [comment]
目前在 Obsidian 上有好几个 vim 相关的插件，其中包括：

- vimrc support
- Add Codemirror matchbrackets.js

感觉现在就缺个find命令的支持了
